generators/add-action/index.js

```diff
@@ -20,13 +20,15 @@ const { sdkCodes } = require('../../lib/constants')
 const sdkCodeToActionGenerator = {
   [sdkCodes.target]: path.join(__dirname, 'target/index.js'),
   [sdkCodes.analytics]: path.join(__dirname, 'analytics/index.js'),
-  [sdkCodes.campaign]: path.join(__dirname, 'campaign-standard/index.js')
+  [sdkCodes.campaign]: path.join(__dirname, 'campaign-standard/index.js'),
+  [sdkCodes.customerProfile]: path.join(__dirname, 'customer-profile/index.js')
 }
 
 const sdkCodeToTitle = {
   [sdkCodes.target]: 'Adobe Target',
   [sdkCodes.analytics]: 'Adobe Analytics',
-  [sdkCodes.campaign]: 'Adobe Campaign Standard'
+  [sdkCodes.campaign]: 'Adobe Campaign Standard',
+  [sdkCodes.customerProfile]: 'Adobe Customer Profile'
```

Comment:
Could you change the label to `Adobe Experience Platform: Realtime Customer Profile` thanks!

Reply:
Updated